app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +349,45 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  parseSpeciesMetricDecimalPlaces = species => {
+    let parsedSpeciesMetrics = Object.assign({}, species);
+    let metricDecimalPlaces = {
+      agg_score: 0,
+      z_score: 1,
+      rpm: 1,
+      count: 0,
+      contigs: 0,
+      contig_r: 0,
+      percent_identity: 1,
+      alignment_length: 1,
+      e_value: 1,
+    };
+
+    Object.keys(species).forEach(outerMetric => {
```

Comment:
We use `count_type` to refer to `nt` and `nr` .

Reply:
If you us `Object.entries(species).foreach((count_type, metrics) => ` instead you get the `count_type` and the `metrics` immediately. Might make it slightly more readable?

Reply:
countType seems a little misleading to me... would metricType be better? and instead of `metrics` would metricValue be better?
`
{
   agg_score: 19195.238841125654
   category: "bacteria"
   common_name: ""
   genus_tax_id: 1827
   max_z_score: 6.475747921182336
   name: "Rhodococcus erythropolis"
   NR: {
      alignment_length: 49
      bg_mean: 39.7436
      bg_stdev: 131.494
      count: 1
      e_value: 21
      percent_identity: 100
      rpm: 891.2655971479501
      z_score: 6.475747921182336
... (NT has same structure of NR)
}
`

Reply:
The formatting isn't so well here but NR and NT are nested within the species object which is why I think count_type is misleading since it also refers to `agg_score: 19195.238841125654
category: "bacteria"
common_name: ""
genus_tax_id: 1827
max_z_score: 6.475747921182336
name: "Rhodococcus erythropolis"`

Reply:
> If you us `Object.entries(species).foreach((count_type, metrics) => ` instead you get the `count_type` and the `metrics` immediately. Might make it slightly more readable?

This is really clean! I didn't know about Object.entries()!

Reply:
`countType` is what we call it across the entire app, so now is not a good time to change it. Also, NT and NR are not metrics types but just the database type we are counting against.`alignment_length`, `rpm` are the metrics..
So:
- `countType` is `nt`, `nr`
- `metrics` is the key `alignment_length`, `rpm`, etc. (and the object with key, values)
- `metricValue` is `6.4757479`, etc.

When you use the entries function it will match adequately.